### PR TITLE
Confirm Donation Picked up

### PR DESCRIPTION
**Pull request by seanpdoyle**
screen shot 2016-04-14 at 5 35 01 pm
screen shot 2016-04-14 at 5 35 09 pm

https://trello.com/c/HG23JmsS

When a cyclist is picking up donations, they can confirm whether or not
the donation has been picked up.

### CODE REVIEW

File: app/controllers/pickups_controller.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+class PickupsController < ApplicationController
+  def destroy
+    @donation = find_donation
+
+    @donation.update!(picked_up: false)
```
Comment (kenyonj) [inline app/controllers/pickups_controller.rb]: Thoughts on not creating an ivar here? It doesn't appear to be needed.

File: app/controllers/pickups_controller.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+    redirect_to checklist_url, flash: { success: t(".success") }
+  end
+
+  def update
+    @donation = find_donation
+
+    @donation.update!(picked_up: true)
```
Comment (kenyonj) [inline app/controllers/pickups_controller.rb]: same here re: ivar

**Comment by seanpdoyle** [comment]
@kenyonj thanks for the review, Justin.

I've pushed an update -- does this look good to go?

**Comment by kenyonj** [comment]
LGTM :ice_cream:

**Pull request merged**